src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt

```diff
@@ -1421,30 +1422,11 @@ TypesMustExist : Type 'System.Data.Common.DbProviderFactories' does not exist in
 TypesMustExist : Type 'System.Data.Common.DbProviderFactoriesConfigurationHandler' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Data.Common.DbProviderFactory.CreatePermission(System.Security.Permissions.PermissionState)' does not exist in the implementation but it does exist in the contract.
 CannotRemoveBaseTypeOrInterface : Type 'System.Data.Common.DbProviderSpecificTypePropertyAttribute' does not implement interface 'System.Runtime.InteropServices._Attribute' in the implementation but it does in the contract.
-TypesMustExist : Type 'System.Data.Odbc.OdbcCommand' does not exist in the implementation but it does exist in the contract.
```

Comment:
Previously the types were missing, now they're present so missing members show up